src/main/java/com/onlinebookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -14,9 +14,9 @@
 public class UserRegistrationRequestDto {
     @NotEmpty @Email @Size(max = 45)
```

Comment:
```suggestion
    @NotBlank @Email @Size(max = 45)
```

Use `@NotBlank` for String variables. Check all places